app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -576,6 +584,36 @@ object TelemetryWrapper {
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.MENU, Value.RELOAD).queue();
     }
 
+    @JvmStatic
```

Comment:
Don't need these then. In general, if you find yourself copying a lot of information that is encapsulated in an API, there should be a better way.